content/ops/contingency-plan.md

```diff
@@ -0,0 +1,122 @@
+
+* The cloud.gov team is able to communicate with each other. Key components and plans to restore this function if disrupted:
+  * We primarily use 18F Slack to communicate among our geographically distributed team. If Slack is unavailable or expected to be unavailable for more than 30 minutes while we're working on a time-sensitive problem (or 6 hours while doing ordinary work), team leads will reach out to team members via GSA email to coordinate use of GSA Google Apps (Gmail, Hangouts, and Drive) as a temporary replacement.
+  * If neither Slack nor GSA Google Apps are available, team leads will coordinate using the GSA phone numbers of team members.
+* The cloud.gov team is able to diagnose and mitigate issues. Key components and plans to restore this function if disrupted:
+  * We need access to the critical platforms and services that cloud.gov relies on: AWS. If we lose access, team members who have access to the credentials and contacts for these systems should work to restore access.
+  * We need to be able to log into our cloud.gov systems, get information from them, and modify them. If we lose access, team members who have access to the credentials for these systems should work to restore access.
+  * We need access to our logging and notification systems. If we lose access, team members who have access to the credentials for these systems should work to restore access.
+  * We need access to our code and documentation, which means access to the information in our 18F GitHub organization. If we lose access to GitHub, each cloud.gov team member has local copies of some cloud.gov git repositories created as part of their ordinary day-to-day work, which can serve as a temporary resource. The 18F Director of Infrastructure makes systematic backups of 18F GitHub repositories at a daily frequency.
+* Our systems are secure (we maintain confidentiality and integrity). Plan to restore this function if disrupted:
+  * If this is threatened, we follow our [Security Incident Response Guide](hhttps://docs.cloud.gov/ops/security-ir).
+* The team can communicate with customers about the status of cloud.gov. Key components and plans to restore this function if disrupted:
+  * If we have more than a small amount of service disruption, team leads will post to our <span class="c10 c6">[externally-hosted status page](https://cloudgov.statuspage.io), post to the 18F Slack #cloud-gov-support channel, and email key contacts at customer teams. If the status page and/or Slack are unavailable, we’ll use GSA email&nbsp;as a backup method to communicate with our key customers. If that’s not possible, we can fall back to phone numbers.
+  * If the disruption is significantly affecting the delivery of our services to customers, team leads will coordinate with the 18F commsquad on how we should communicate this externally, including potentially using @18F Twitter and later following up with an 18F blog post.
```

Comment:
I think that for now we should have our own roles...

On Fri, Jul 1, 2016 at 10:06 AM, Aidan Feldman <notifications@github.com>
wrote:

> In content/ops/contingency-plan.md
> <https://github.com/18F/cg-docs/pull/297#discussion_r69313315>:
>
> > +
> > +These are essential functions of our cloud.gov information systems, listed in order of which would need to be restored first in case of critical problems with more than one function:
> > +
> > +* The cloud.gov team is able to communicate with each other. Key components and plans to restore this function if disrupted:
> > +  * We primarily use 18F Slack to communicate among our geographically distributed team. If Slack is unavailable or expected to be unavailable for more than 30 minutes while we're working on a time-sensitive problem (or 6 hours while doing ordinary work), team leads will reach out to team members via GSA email to coordinate use of GSA Google Apps (Gmail, Hangouts, and Drive) as a temporary replacement.
> > +  * If neither Slack nor GSA Google Apps are available, team leads will coordinate using the GSA phone numbers of team members.
> > +* The cloud.gov team is able to diagnose and mitigate issues. Key components and plans to restore this function if disrupted:
> > +  * We need access to the critical platforms and services that cloud.gov relies on: AWS. If we lose access, team members who have access to the credentials and contacts for these systems should work to restore access.
> > +  * We need to be able to log into our cloud.gov systems, get information from them, and modify them. If we lose access, team members who have access to the credentials for these systems should work to restore access.
> > +  * We need access to our logging and notification systems. If we lose access, team members who have access to the credentials for these systems should work to restore access.
> > +  * We need access to our code and documentation, which means access to the information in our 18F GitHub organization. If we lose access to GitHub, each cloud.gov team member has local copies of some cloud.gov git repositories created as part of their ordinary day-to-day work, which can serve as a temporary resource. The 18F Director of Infrastructure makes systematic backups of 18F GitHub repositories at a daily frequency.
> > +* Our systems are secure (we maintain confidentiality and integrity). Plan to restore this function if disrupted:
> > +  * If this is threatened, we follow our [Security Incident Response Guide](hhttps://docs.cloud.gov/ops/security-ir).
> > +* The team can communicate with customers about the status of cloud.gov. Key components and plans to restore this function if disrupted:
> > +  * If we have more than a small amount of service disruption, team leads will post to our <span class="c10 c6">[externally-hosted status page](https://cloudgov.statuspage.io), post to the 18F Slack #cloud-gov-support channel, and email key contacts at customer teams. If the status page and/or Slack are unavailable, we’ll use GSA email&nbsp;as a backup method to communicate with our key customers. If that’s not possible, we can fall back to phone numbers.
> > +  * If the disruption is significantly affecting the delivery of our services to customers, team leads will coordinate with the 18F commsquad on how we should communicate this externally, including potentially using @18F Twitter and later following up with an 18F blog post.
>
> Maybe "the 18F Communications team (@commsquad in Slack)"?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/cg-docs/pull/297/files/fbf9f3e6f123bbcf1ca9001269926fa6e9ab7e3b#r69313315>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAR_AgDXUGf0UIYU94odggdAU1JRSZUks5qRSzugaJpZM4I_vmx>
> .
>



-- 
Diego Lapiduz
Director, cloud.gov
diego.lapiduz@gsa.gov
512 766 3434